src/BlazorWasmSdk/Tasks/Microsoft.NET.Sdk.BlazorWebAssembly.Tasks.csproj

```diff
@@ -63,6 +64,9 @@
 
   <ItemGroup>
     <Compile Include="**\*.cs" />
+    <!-- C:\work\dotnet-sdk\src\StaticWebAssetsSdk\Tasks\Data\ -->
+    <!-- C:\work\dotnet-sdk\src\BlazorWasmSdk\Tasks\ -->
+    <Compile LinkBase="StaticWebAssets" Include="..\..\StaticWebAssetsSdk\Tasks\Data\**\StaticwebAsset.cs" />
```

Comment:
Linked static web assets so that we can use them inside the build tasks instead of using the item groups